nvflare/app_common/workflows/scatter_and_gather.py

```diff
@@ -41,10 +41,10 @@ def _check_non_neg_int(data: Any, name: str):
 class ScatterAndGather(Controller):
     def __init__(
         self,
-        min_clients: int = 1,
+        min_clients: int = 1000,
         num_rounds: int = 5,
         start_round: int = 0,
-        wait_time_after_min_received: int = 10,
+        wait_time_after_min_received: int = 0,
```

Comment:
Should we be changing this default value?

Reply:
I think better the default is 0, so default it will not wait.

Reply:
or you like 10 better?

Reply:
The idea here is that even when setting `min_clients < len(targets)`, some clients can still contribute to the global model if they contribute within `wait_time_after_min_received`. It's a useful feature in real-world FL deployment. It's also about backward compatibility at this stage. Should we change the default behavior here? Either way, I don't have a strong feeling about this. @yanchengnv to comment.

Reply:
I'm not sure either. But it's good to keep default the original value, since there might be people that are relying on this.